MRETestBed/Assets/TestBed Assets/Scripts/MREComponent.cs

```diff
@@ -230,13 +230,13 @@ private void MREApp_OnConnectFailed(MixedRealityExtension.IPC.ConnectFailedReaso
 		}
 	}
 
-	private void MRE_OnUserJoined(IUser user)
+	private void MRE_OnUserJoined(IUser user, bool isLocalUser)
```

Comment:
>, bool isLocalUser [](start = 41, length = 18)

Is this just the result of the test bed not being updated from a previous change?  Only because it seems unrelated to this pull request.

Reply:
Correct, we just needed this to get it to compile.

---
In reply to: [473272617](https://github.com/microsoft/mixed-reality-extension-unity/pull/229#discussion_r473272617) [](ancestors = 473272617)